core-common/src/main/java/eu/arrowhead/common/CoreDefaults.java

```diff
@@ -89,6 +89,13 @@
 	public static final int DEFAULT_QOS_DEFAULT_REFERENCE_LOST_PER_MEASUREMENT_PERCENT = 0;
 	public static final boolean DEFAULT_QOS_ENABLED_RELAY_TASK = false;
 	public static final boolean DEFAULT_QOS_IS_GATEKEEPER_PRESENT = false;
+	public static final String DEFAULT_QOS_MONITOR_PROVIDER_TYPE = "defaultexternal";
+	public static final String DEFAULT_QOS_MONITOR_PROVIDER_NAME = "qosmeasurer";
+	public static final String DEFAULT_QOS_MONITOR_PROVIDER_ADDRESS = "127.0.0.1";
+	public static final int DEFAULT_QOS_MONITOR_PROVIDER_PORT = 8888;
+	public static final String DEFAULT_QOS_MONITOR_PROVIDER_PATH = "/ping-icmp";
+	public static final boolean DEFAULT_QOS_MONITOR_PROVIDER_SECURE = true;
+	public static final String DEFAULT_QOS_MONITOR_PROVIDER_AUTHINFO = "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA+5IuB4MwgKvroJ0LCJu4DdhHno77LHkTW+ufF7+HVOCRgi2BKCtR4PNDkov04pmcjljit8viqnVqca2S7c9YSS+20NnxK5nrql/VYdIg98wakErsE40iP8QI7arxGstn2PP1CFfFx4CzbMWRprr9BvDpD8rwqdeOMPrUzWRlCTpvWDcBUO84CbhAUR1v77MCjPu/DjO26yufQWAaglGpbGy980JtzLZ0BmZzQK929MdD5PBgveKJGz7NA3jALidrVspLkduG25az3zG//1mm6ybwVd3dZuogLlscSAszEeexkfZGUYrQZ6XS0cMKHzhJULRQiU8Rud94zfc3HLPL6QIDAQAB";
```

Comment:
don't hardcode the public key into the code. The empty string is a good default. I really don't understand why do we need this.

Reply:
removed